glances/plugins/containers/engines/docker.py

```diff
@@ -215,25 +215,26 @@ class DockerContainersExtension:
 
     CONTAINER_ACTIVE_STATUS = ['running', 'paused']
 
-    def __init__(self):
+    def __init__(self, server_urls:str|list = 'unix:///var/run/docker.sock'):
```

Comment:
Same remark, let the server_urls default value to '' (empty).

Reply:
Just something to note with Python support.

Glances v4 would be target Py3.8+

The `str |  list` syntax was only support from Py3.10 onwards, iirc.
So better use the older Union[] style to prevent older interpreters from crashing.

Also I think returning a list[str] even when there is only a single elemenet would reduce unnecessary type checks later